### ISSUE TITLE
No Audio Generation??.

### ISSUE THREAD

**Issue by amssss0** [edwko/OuteTTS#36]
hello thanks for Open sourcing the project. 
i am having a little issue running it, here is the code i used and the output and i also attacked the output audio, that is the problem, every run i get no output oudio

import outetts

# Configure the model
model_config = outetts.HFModelConfig_v1(
    model_path="OuteAI/OuteTTS-0.2-500M",
    language="en",  # Supported languages in v0.2: en, zh, ja, ko
    verbose=True,
)

# Initialize the interface
interface = outetts.InterfaceHF(model_version="0.2", cfg=model_config)
# Optional: Load speaker from default presets
interface.print_default_speakers()
speaker = interface.load_default_speaker(name="female_1")

output = interface.generate(
    text="Before using these pre-trained models, you agree to inform the listeners that the speech samples are synthesized by the pre-trained models, unless you have the permission to use the voice you synthesize. That is, you agree to only use voices whose speakers grant the permission to have their voice cloned, either directly or by license before making synthesized voices public, or you have to publicly announce that these voices are synthesized if you do not have the permission to use these voices.",
    # Lower temperature values may result in a more stable tone,
    # while higher values can introduce varied and expressive speech
    temperature=0.1,
    repetition_penalty=1.1,
    max_length=4096,
    speaker=speaker,
)

output.save("output.wav")

OUTPUT:
making attention of type 'vanilla' with 768 in_channels
2024-11-27 15:15:03.702 | INFO     | outetts.version.v1.interface:generate:228 - Input tokens: 1358
2024-11-27 15:15:03.702 | INFO     | outetts.version.v1.interface:generate:229 - Generating audio...
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation.

=== ALL AVAILABLE SPEAKERS ===
Total: 16 speakers across 4 languages
--------------------------------------------------

EN (6 speakers):
  - male_1
  - male_2
  - male_3
  - male_4
  - female_1
  - female_2

JA (4 speakers):
  - male_1
  - female_1
  - female_2
  - female_3

KO (4 speakers):
  - male_1
  - male_2
  - female_1
  - female_2

ZH (2 speakers):
  - male_1
  - female_1

=== SPEAKERS FOR CURRENT INTERFACE LANGUAGE ===
Language: EN (6 speakers)
--------------------------------------------------
  - male_1
  - male_2
  - male_3
  - male_4
  - female_1
  - female_2

To use a speaker: load_default_speaker(name)

2024-11-27 15:16:27.268 | INFO     | outetts.version.v1.interface:generate:241 - Audio generation completed
2024-11-27 15:16:27.269 | WARNING  | outetts.version.v1.interface:play:90 - Audio playback is disabled. Check sounddevice installation.

OK lets try again:
2024-11-27 15:17:44.355 | INFO     | outetts.version.v1.interface:generate:228 - Input tokens: 1252
2024-11-27 15:17:44.355 | INFO     | outetts.version.v1.interface:generate:229 - Generating audio...
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation.
2024-11-27 15:19:08.050 | WARNING  | outetts.version.v1.interface:get_audio:129 - No audio tokens found in the output
2024-11-27 15:19:08.051 | INFO     | outetts.version.v1.interface:generate:241 - Audio generation completed
2024-11-27 15:19:08.051 | WARNING  | outetts.version.v1.interface:save:79 - Audio is empty, skipping save.

here is what i got from both runs, empty audio because "No audio tokens found in the output".
[output.zip](https://github.com/user-attachments/files/17932488/output.zip)

i think the output is empty here:
output = self.model.generate(
      input_ids=input_ids,
      config=GenerationConfig(
          temperature=temperature,
          repetition_penalty=repetition_penalty,
          max_length=max_length
      )
  )
https://github.com/edwko/OuteTTS/blob/main/outetts/version/v1/interface.py#L231
not sure why, do anyone have any idea on whats going on??

**Comment by edwko** [comment]
It seems there might be an issue with the installation. 

```plaintext
2024-11-27 15:16:27.269 | WARNING | outetts.version.v1.interface:play:90 - Audio playback is disabled. Check sounddevice installation.
```

`sounddevice` should be installed by outetts package.

```plaintext
2024-11-27 15:17:44.355 | INFO | outetts.version.v1.interface:generate:228 - Input tokens: 1252
...
2024-11-27 15:19:08.050 | WARNING | outetts.version.v1.interface:get_audio:129 - No audio tokens found in the output
```

This suggests that the model didn’t generate any audio tokens, despite having sufficient input tokens. With a limit of 4096, the input length should leave plenty of space for generation.

Try reinstalling the package in a fresh environment to ensure all dependencies are up to date, and then give it another shot.

**Comment by amssss0** [comment]
thanks, i will try installing sounddevice.

**Comment by amssss0** [comment]
still:
2024-12-01 15:49:56.695 | INFO     | outetts.version.v1.interface:generate:259 - Input tokens: 1252
2024-12-01 15:49:56.696 | INFO     | outetts.version.v1.interface:generate:260 - Generating audio...
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation.
2024-12-01 15:51:19.497 | WARNING  | outetts.version.v1.interface:get_audio:135 - No audio tokens found in the output
2024-12-01 15:51:19.498 | INFO     | outetts.version.v1.interface:generate:275 - Audio generation completed
2024-12-01 15:51:19.498 | WARNING  | outetts.version.v1.interface:save:84 - Audio is empty, skipping save.
2024-12-01 15:51:19.499 | WARNING  | outetts.version.v1.interface:play:91 - Audio is empty, skipping playback.
